src/java.desktop/share/classes/javax/swing/ImageIcon.java

```diff
@@ -207,6 +214,7 @@ public ImageIcon (URL location) {
      * Creates an ImageIcon from the image.
      * @param image the image
      * @param description a brief textual description of the image
+     * @throws {@code NullPointerException} if (@code null) Image is passed.
```

Comment:
```suggestion
     * @throws NullPointerException if a {@code null} image is passed
```
Here, image shouldn't be capitalised, you mean it as the literal _“image”_ rather than a type.

I still prefer specifying the parameter name:
```suggestion
     * @throws NullPointerException if {@code image} is {@code null}
```

Reply:
ok